<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ChetnaM-PC" ignored="0" name="suite17" tests="3" failures="1" timestamp="24 May 2017 15:46:30 GMT" time="1730.902" errors="0">
  <testcase name="verify" time="607.361" classname="com.adv.qa.selenium.tests.currency.phase2.A084_Authorisation_Value_Levels_DefinitionTest"/>
  <testcase name="verify" time="800.031" classname="com.adv.qa.selenium.tests.currency.phase2.A085_Authorisation_By_Value_LevelTest">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.adv.qa.selenium.tests.currency.phase2.A085_Authorisation_By_Value_LevelTest.verify() didn&amp;apos;t finish within the time-out 800000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.adv.qa.selenium.tests.currency.phase2.A085_Authorisation_By_Value_LevelTest.verify() didn't finish within the time-out 800000
	at java.lang.Thread.sleep(Native Method)
	at com.adv.qa.selenium.helpers.WaitHelper.waitAdditional(WaitHelper.java:245)
	at com.adv.qa.selenium.framework.pageObjects.currency.CurrencyPage.enterValueLevelAuthForPM(CurrencyPage.java:8160)
	at com.adv.qa.selenium.tests.currency.phase2.A085_Authorisation_By_Value_LevelTest.createStructureAuthorisers(A085_Authorisation_By_Value_LevelTest.java:113)
	at com.adv.qa.selenium.tests.currency.phase2.A085_Authorisation_By_Value_LevelTest.verify(A085_Authorisation_By_Value_LevelTest.java:94)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- verify -->
  <testcase name="verify" time="207.304" classname="com.adv.qa.selenium.tests.currency.phase2.A089_Amend_AP_Company_Controls_For_AuthorisationTest"/>
</testsuite> <!-- suite17 -->
